lib/callback_model.js

```diff
@@ -5,7 +5,6 @@
 'use strict';
 
 var defs = require('./defs');
-var Promise = require('bluebird');
 var inherits = require('util').inherits;
```

Comment:
Unrelated to this PR but i just notices inherits, NodeJS docs mention discourage use of inherits and that class extends should be used instead.

Reply:
I'll open PR for this